scripts/participateInAuction.js

```diff
@@ -96,20 +99,24 @@ async function unlockMgnAndTokens({
 }
 
 
-async function claimMgn({ poolNumber, pool, state, infoAuction }) {
+async function claimMgn({ poolNumber, pool, mgn, state, infoAuction, debugAuction }) {
   // We could check the expire time easily of the MGN, but we will take the simple approach of
   // executing "withdrawUnlockedMagnoliaFromDx" if the call succeeds
```

Comment:
```suggestion
```

Reply:
Thanks for the review. I removed also the other comment. I think it was not necessary either